src/libsync/vfs/xattr/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-if (UNIX)
+if (LINUX)
     set(vfs_xattr_SRCS vfs_xattr.cpp)
     if (APPLE)
```

Comment:
Why not removing that branch? I can not see that we have this file. And I don't see that APPLE would be ever true after LINUX was true.

Reply:
indeed, I kept my changes minimal but I can do it and check later if I find this damn file (or ask Kevin)